src/components/Nav.jsx

```diff
@@ -0,0 +1,46 @@
+import logo from "../assets/panda-logo.png";
+import brandName from "../assets/판다마켓.png";
+import avatar from "../assets/avatar.png";
+import { Link, useLocation } from "react-router-dom";
+
+export default function Nav() {
+  const location = useLocation();
+  const pathname = location.pathname;
+
+  return (
+    <nav className="border-b border-b-gray-300">
+      <div
+        className="
+          max-w-[1920px]
+          mx-auto
+          px-[2rem] sm:px-[5rem] md:px-[10rem] lg:px-[20rem]
+          py-4
+          flex justify-between
+        "

```

Comment:
❗️ 수정요청
tailwind 사용시 한줄로 적으시는 것이 권장되는 방식 입니다~
```suggestion
        className="max-w-[1920px] mx-auto px-[2rem] sm:px-[5rem] md:px-[10rem] lg:px-[20rem] py-4 flex justify-between"
```